services/core-web/src/tests/components/Forms/parties/__snapshots__/AddFullPartyForm.spec.js.snap

```diff
@@ -19,7 +19,7 @@ exports[`AddFullPartyForm renders properly 1`] = `
         <div
           className="center margin-large"
         >
-          <Memo(RadioGroup)
+          <Memo([object Object])
```

Comment:
The [object Object] looked more off than the other snapshot updates- I verified that this component still looks the same as it does on test.